rct229/rulesets/ashrae9012019/data/ashrae_90_1_table_G3_9_1.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
If the table is not needed, then we don't add it.
But I think we can switch the table in G3.9.1 data function to this json file.

Reply:
Weili, 

Could we use the ```ashrae_90_1_prm_2019.motors.json``` file as it was and delete the ```ashrae_90_1_table_G3_9_1.json``` because the ```ashrae_90_1_prm_2019.motors.json``` file has more data?

Reply:
Weili and I discussed this offline, and we decided to use the ```ashrae_90_1_table_G3_9_1.json```.